packages/ckeditor5-engine/src/model/document.ts

```diff
@@ -270,7 +286,7 @@ export default class Document {
 	 *
 	 * @param {Function} postFixer
 	 */
-	registerPostFixer( postFixer ) {
+	public registerPostFixer( postFixer: ( writer: Writer ) => unknown ): void {
```

Comment:
Post-fixer callbacks should return `true` if they modified the model. Maybe we could be more verbose here?